backend/compact-connect/docs/design/README.md

```diff
@@ -2,7 +2,33 @@
 
 Look here for continued documentation of the back-end design, as it progresses.
 
+## Table of Contents
+- **[Compacts and Jurisdictions](#compacts-and-jurisdictions)**
+- **[License Ingest](#license-ingest)**
+- **[User Architecture](#user-architecture)**
+
+## Compacts and Jurisdictions
+
+The CompactConnect system supports multiple licensure compacts and, within each compact, multiple jurisdictions. The
+jurisdictions it supports within each compact is all 50 states, Washington D.C., Puerto Rico, and the Virgin Islands.
+
+### Adding a compact to CompactConnect
+
+When a new compact joins CompactConnect, some configuration has to be done to add them to the system. First, a new
+entry has to be added to the list of supported compacts, found in [cdk.json](../../cdk.json). Each compact is
+represented there with an abbreviation, which determines how the compact will be represented in the API as well as
+in its corresponding Oauth2 access scopes. Because of the way that the scopes are represented, the compact abbreviation
+must not overlap with any jurisdiction abbreviations (which correspond to the jurisdictions' USPS postal abbreviations).
+**Since postal abbreviations are all two letters, make a point to choose a compact abbreviation that is at least four
```

Comment:
Perhaps add a quick note to say, "The compact abbreviation does not need to match the one used by CSG." (Or whatever those alternative abbreviations are used for.)